deploy/manifests/dev/us-east-2/cluster/kube-system/aws-auth.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: aws-auth
+data:
+  mapRoles: |
+    - groups:
+      - system:bootstrappers
+      - system:nodes
+      rolearn: arn:aws:iam::407967248065:role/default_node_group-eks-node-group-20220326104849172600000002
```

Comment:
Did you manually create this? or what are the steps involved in updating this?

Reply:
Related to the [comment](https://github.com/filecoin-project/storetheindex/pull/293#discussion_r837221587) I made earlier; this is created and managed by EKS as part of EKS managed node group. 

It is not created via UI clicking if that's we mean by manual. The EKS managed node group is defined and applied via terraform [here](https://github.com/filecoin-project/storetheindex/blob/65af0f4b1b5272ad407d90b9afa7b06d1d36e146/deploy/infrastructure/dev/us-east-2/eks.tf#L15-L24).